com.unity.netcode.gameobjects/Tests/Runtime/RpcTests.cs

```diff
@@ -62,7 +62,6 @@ public void MyNativeListServerRpc(NativeList<ulong> clientId, ServerRpcParams pa
             }
 #endif
 
-
```

Comment:
You could remove the file from this PR by adding the CR/LF back.  (_nit-picky of me_)

Reply:
When I save it in Rider, it just removes the CR/LF again. 😔